src/app/core/testing/mock-enclosure/enclosure-templates/utils/make-array-device-slots.utils.ts

```diff
@@ -18,15 +18,15 @@ export function makeArrayDeviceSlots(
       drive_bay_number: slot,
       descriptor: `slot${slot.toString().padStart(2, '0')}`,
       status: EnclosureStatus.Ok,
-      dev: null,
+      dev: null as null,
       supports_identify_light: options.supportsIdentifyLight ?? false,
-      drive_bay_light_status: null,
-      size: null,
-      model: null,
-      serial: null,
-      type: null,
-      rotationrate: null,
-      pool_info: null,
+      drive_bay_light_status: null as null,
+      size: null as null,
+      model: null as null,
+      serial: null as null,
+      type: null as null,
+      rotationrate: null as null,
+      pool_info: null as null,
```

Comment:
`null as null` means that only `null` is allowed as a value, which is not true. It's better to case `as DashboardEnclosureSlot` on the object.